tasks/dependencies.yml

```diff
@@ -36,6 +36,16 @@
         VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
       when: (not ansible_check_mode) and conditional_dependencies.stdout_lines | length > 0
 
+    - name: Install additional packages into galaxy's virtual environment
+      pip:
+        name: "{{ galaxy_additional_venv_packages }}"
+        virtualenv: "{{ galaxy_venv_dir }}"
+        virtualenv_command: "{{ galaxy_virtualenv_command | default(pip_virtualenv_command | default(omit)) }}"
+      environment:
+        PYTHONPATH: null
+        VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
+      when: galaxy_additional_venv_packages is not []
```

Comment:
maybe this check could be `when: galaxy_additional_venv_packages`, I think that should be falsey?

Reply:
Thanks!  I'm not sure why I thought that `when` would work.

Good CI btw!